src/main/java/eu/xenit/gradle/docker/DockerBuildExtension.java

```diff
@@ -1,7 +1,11 @@
 package eu.xenit.gradle.docker;
 
+import static com.bmuschko.gradle.docker.shaded.org.apache.commons.lang.StringUtils.leftPad;
```

Comment:
This is internal code of a dependency

Reply:
Okay, ill change it. Using that one seemed cleaner since it wouldnt be adding a 'new' dependency for something that is already present

Reply:
I assume we're also not allowed to use the gradle internal dep

Reply:
As there is no guarantee that those dependencies will still exist at any time, no.
Especially external dependencies that have been shaded in (like you can see here with the  `shaded` package name, or in gradle with `impldep`)